integration/airflow/openlineage/airflow/macros.py

```diff
@@ -21,17 +19,20 @@ def lineage_run_id(task: "BaseOperator", task_instance: "TaskInstance"):
     PythonOperator(
         task_id='render_template',
         python_callable=my_task_function,
-        op_args=['{{ lineage_run_id(task, task_instance) }}'], # lineage_run_id macro invoked
+        op_args=['{{ lineage_run_id(task_instance) }}'], # lineage_run_id macro invoked
         provide_context=False,
         dag=dag
     )
     """
     return OpenLineageAdapter.build_task_instance_run_id(
-        task_instance.dag_id, task.task_id, task_instance.execution_date, task_instance.try_number
+        task_instance.dag_id,
```

Comment:
Why `dag_id` is an attribute of `task_instance`? Shouldn't it be a `task` attribute?
https://airflow.apache.org/docs/apache-airflow/2.6.3/_api/airflow/models/taskinstance/index.html#airflow.models.taskinstance.TaskInstance

Reply:
I think the task_instance inherits the dag_id from task: https://github.com/apache/airflow/blob/0e8f108313d4af0b450581661aeb8ed35e82a8e6/airflow/models/taskinstance.py#L1432 ?

Reply:
Why not use `task_instance.task_id` when?

Reply:
Naming consistencies yeah :smile: 

IMHO I'd keep this line as it is - otherwise `build_task_instance_run_id` would take `task_instance.task_id` and `task_instance.task.task_id` as arguments, which IMHO would be even more confusing.